spec/exe/metasploit_cache_spec.rb

```diff
@@ -257,6 +262,259 @@ def self.each_reference_name(max_run_count: nil, module_path_real_pathname:, mod
                 )
               }
             end
+
+            context '(staged)' do
+              payload_staged_reference_names = %w{
+                  android/meterpreter/reverse_http
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.